ape.toml.template

```diff
@@ -17,9 +17,21 @@ timeout = ${AGENT_TIMEOUT}
 # debug level can be info|warning,warn|error,err|debug
 log_level = "${AGENT_LOG_LEVEL}"
 
+# Basic HTTP authentication for '/metrics'.
 basic_auth_username="${BASIC_AUTH_USERNAME}"
 basic_auth_password="${BASIC_AUTH_PASSWORD}"
 
+# Metrics Whitelist - If specified, only these metrics will be scraped. An empty list will exclude all metrics.
+
+# Namespace metrics whitelist
+# namespace_metrics_whitelist=[${NAMESPACE_METRICS_WHITELIST}]
```

Comment:
We need a default whitelist with all the metrics for independent installations (outside of k8t) as well. In that whitelist, the unselected items should be commented out so the totality of selectable items is immediately visible.

ps. It seems like by default the whitelist is disabled. In that case, please disregard this comment, but add a comment that says commenting out the config key will disable the whitelist all together.

Reply:
Yes, by default the whitelist is disabled. I'll improve the comments in the config file.